src/main/java/edu/wpi/first/wpilibj/Joystick.java

```diff
@@ -0,0 +1,9 @@
+package edu.wpi.first.wpilibj;
+

```

Comment:
The Joystick class should be part of the edu.wpi.first.wpilibj library and a separate package shouldn't be created. I'm assuming that the package was created due to issues in compiling code on a restricted school network. The package should be deleted and use the edu.wpi.first.wpilibj.